insights/tools/insights_inspect.py

```diff
@@ -61,7 +61,7 @@
 
 >>> insights-inspect examples.rules.bash_version.report
 IPython Console Usage Info:
-Enter 'make_pass.' and tab to get a list of properties
+Enter 'Hostname.' and tab to get a list of properties
```

Comment:
This should actually be ...
Enter 'report.'
Line 66 should be 'report.<property_name>' instead of make_pass.<property_name>
Line 76 below should be 'report.values()' instead of 'make_pass.values()'
Before line 85 there should be a line with 'import insights'

Reply:
Updated